chapter06/ajd/exercise2.pl

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use File::Slurp;
+use JSON;
+
+# Get the file read
+my %total_bytes;
+
+# Import the old file - JSON version
+my $json = new JSON();
+my $data_file = 'logdata.json';
+
+if ( -s $data_file ) {
+  # Needed to force scalar context as otherwise File::Slurp returns an array of lines
+  %total_bytes = %{ $json->decode( scalar ( read_file( $data_file ) ) ) };
```

Comment:
This feels like too many parentheses, I'd find it more readable as `$json->decode( scalar read_file( $data_file ) )`.